src/utils/logger.rs

```diff
@@ -202,6 +222,11 @@ mod logger_thread {
                     flush_logs_to_stderr(&mut buffer);
                     break;
                 }
+                Message::Flush { finished_barrier } => {
+                    flush_logs_to_stderr(&mut buffer);
+                    finished_barrier.wait();
+                    break;
```

Comment:
oops this "break" shouldn't be here

https://github.com/ouch-org/ouch/pull/748